e2e_testing/torchscript/basic.py

```diff
@@ -738,3 +738,35 @@ def forward(self, input, tensor1, tensor2):
 @register_test_case(module_factory=lambda: AddCDivModule())
 def AddCDivModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(1,3), tu.rand(1,3), tu.rand(1,3))
+
+class Fill_TensorModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.float32, True),
+    ])
+    def forward(self, tensor):
+        return torch.ops.aten.fill_(tensor, 3.0)
+
+@register_test_case(module_factory=lambda: Fill_TensorModule())
+def Fill_TensorModule_basic(module, tu: TestUtils):
+    module.forward(torch.randn(3, 2, 4))
+
+class Fill_TensorModule_1(torch.nn.Module):
```

Comment:
nit: change the test name to something like `Fill_TensorFloat64_ScalarFloat64` and the above `Fill_TensorFloat32_ScalarFloat64`

Reply:
Can you also add another test `Fill_TensorInt_ScalarFloat64`?

Reply:
Done! Added. Thanks.